action.yaml

```diff
@@ -96,6 +100,14 @@ runs:
                   body: issue_body,
               })
           } else if (${{ steps.get-id.outputs.result == -1 && inputs.post-to-issue }}) {
+            if (${{ inputs.append-to-issue }}) {
```

Comment:
This needs to be in the if block when `result != -1` otherwise there is no issue to comment on. Looks good otherwise!

Reply:
Ah yeah, thanks, all fixed now.